### PR TITLE
Fixes issue no 11442 | is_EmptySet returns only True or False now

### PR DESCRIPTION
**Pull request by ashutoshsaboo**
Solves and Fixes Issue #11442 . 

is_EmptySet now only returns True or False, and not None. Solves Issue #11442 .

Please review this @jksuom @asmeurer @certik . :smile:

### CODE REVIEW

File: sympy/sets/tests/test_sets.py [hunk 1]
```diff
@@ -1052,3 +1052,7 @@ def test_issue_11174():
 
     soln = Intersection(S.Reals, FiniteSet(x), evaluate=False)
     assert Intersection(FiniteSet(x), S.Reals) == soln
+
+def test_issue_11442():
+    assert Interval(2, 7).is_EmptySet is False
+    assert Interval(2, 1).is_EmptySet is False
```
Comment (asmeurer) [inline sympy/sets/tests/test_sets.py]: You need a newline at the end of this file.

**Comment by asmeurer** [comment]
Why are some of the properties set to None and other set to False? It seems that they all should be one or the other.

**Comment by ashutoshsaboo** [comment]
@asmeurer I am sorry. What do you mean to say? 
I changed `is_EmptySet` to be equal to `False` by default instead of `None` which causes this issue, since once the empty set function recognizes the set as an empty set, then it gets set to True, otherwise it is always False. There's no option of being that equal to `None`, that's the reason. 

Apart from that all other properties are set to the by-default values, that existed till now only. I haven't changed them. @asmeurer 

Also any reason as to why the tests failed?

File: sympy/sets/tests/test_sets.py (hunk 1 above)
Comment (ashutoshsaboo) [inline sympy/sets/tests/test_sets.py]: @asmeurer Added the blank line in the end in the recent commit. :+1:

**Comment by ashutoshsaboo** [comment]
@asmeurer Resolved this. Was a problem with the test for this issue that I wrote. Tests pass locally now, so must pass on Travis as well. :+1:

**Comment by asmeurer** [comment]
I'm unclear if this is the right fix. Should these things default to None or False? Wouldn't another way to fix this be to set is_EmptySet on Interval? Is None meaningful for this `is_` properties?

**Comment by siefkenj** [comment]
Uniqueness of the empty set rears its ugly head...  The `is_CapitalLetter` should refer to whether something is an instance of a class or pseudo-class and so should be `True` or `False`.  I don't think it makes much sense to have an `is_EmptySet` at all...though an `is_empty` property would make sense for sets.  Is `is_EmtpySet` actually used in the code for anything other than `is_empty`?

**Comment by smichr** [comment]
The added tests evaluate like this in master and None is returned if it is not known if a set is empty or not:
```python
>>> Interval(1,2).is_EmptySet
False
>>> Interval(2,1).is_EmptySet
True
>>> Interval(x,y).is_EmptySet is None
True
```

**Comment by siefkenj** [comment]
Again, I think `is_CapitalLetter` returning `None` makes the API inconsistent...

**Comment by jksuom** [comment]
There could be another property `is_empty` that could return `None` for `Interval(x, y)`. I think that `is_EmptySet` should be `True` only for `EmptySet` and `False` otherwise. (Then it would, in fact, be redundant as `is EmptySet` would have the same meaning.) So I suggest that `is_EmptySet` be replaced by `is_empty` everywhere.

**Comment by oscarbenjamin** [comment]
I agree. I'm going to start an issue from your last comment @jksuom

**Pull request closed without merging**